### ISSUE TITLE
make weapons have a flash whenshot

### ISSUE THREAD

**Issue by Lightning38088** [yogstation13/yogstation#2057]
Weapons which use bullets use a regular type of flash
while eguns should make a small flash the color of the intent like red for harm and blue for disable
im not sure if tasers should emit a light tho when shot.

**Comment by McDonald072** [comment]
flying lasers and tasers should give off light for cool factor

**Comment by Lightning38088** [comment]
they do give off light when shot (the beams) its just no colored.

**Comment by AdamElTablawy** [comment]
why would a laser emit a flash of light when the fucking projectile is a flash of light

the 'muzzle flash' you are imagining IS THE FUCKING PROJECTILE

well tbh i don't know what the fuck ss13 lasers are since they have a travel time so i'm going with phlebotinum

**Comment by Mrcreepssss** [comment]
yeah, lasers shouldn't have a muzzle flash, though other weapons should!
